src/clib/pio_nc.c

```diff
@@ -1869,6 +1958,37 @@ int PIOc_inq_att(int ncid, int varid, const char *name, nc_type *xtypep,
     }
 #endif
 
+#ifdef _HDF5
+    if (file->iotype == PIO_IOTYPE_HDF5)
+    {
+        ierr = PIO_ENOTATT;
+        for (int i = 0; i < file->hdf5_num_attrs; i++)
+        {
+            if (!strcmp(name, file->hdf5_attrs[i].att_name) &&
```

Comment:
In future PR don't forget to replace strcmps by strncmps

Reply:
OK, we can also use strncmps for ADIOS type.